<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMaterialPriceWindow</class>
 <widget class="QDialog" name="AddMaterialPriceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>233</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>227</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>233</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Side Iron Price</string>
  </property>
  <layout class="QVBoxLayout" name="addMaterialPriceVerticalLayout">
   <item>
    <widget class="QLabel" name="AddPriceTitleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Side Iron Price</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DynamicComboBox" name="materialNameComboBox"/>
   </item>
   <item>
    <widget class="QWidget" name="formLayout" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="widthLabel">
        <property name="text">
         <string>Width :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="widthEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lengthLabel">
        <property name="text">
         <string>Length :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lengthEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="priceLabel">
        <property name="text">
         <string>Price :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="priceEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="priceTypeLabel">
        <property name="text">
         <string>Per :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="priceTypeComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="acceptButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DynamicComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/DynamicComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>acceptButtons</sender>
   <signal>accepted()</signal>
   <receiver>AddMaterialPriceWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButtons</sender>
   <signal>rejected()</signal>
   <receiver>AddMaterialPriceWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
